docs/site/downloads/cldr-48.md

```diff
@@ -292,6 +292,9 @@ For a full listing, see [Transforms Delta].
 }
 ```
 
+- The `rgScope` values are now approved, but they are not output in cldr-json at present.
+See [CLDR-19029] for a discussion of the issue.
+
 The `ar-OrdinalRules.txt` file contains all rules for this locale:
```

Comment:
@srl295 This change is on the maint/maint-48 branch but the cldr-48.md that we publish is the one on main, can you make this change on that one? Though I guess my cherry-pick will take it over there, so maybe it is not worth it, unless it causes a CI test failure or something.